### PR TITLE
 add APT43 + tools 

### PR DESCRIPTION
**Pull request by Delta-Sierra**

### CODE REVIEW

File: clusters/tool.json [hunk 1]
```diff
@@ -9138,7 +9157,7 @@
       },
       "related": [],
       "uuid": "be419332-61ab-45f0-979e-56f78a223c8e",
-      "value": "metasploit"
+      "value": "METASPLOIT is a penetration testing framework whose features include vulnerability testing, network enumeration, payload generation and execution, and defense evasion. Availability: Public"
```
Comment (cvandeplas) [inline clusters/tool.json:9160]: Shouldn't this be in the "description" field instead?

Comment (Delta-Sierra) [reply clusters/tool.json:9160]: Woops indeed I'll fix that. thanks for catching it!

**Comment by cvandeplas** [comment]
Nice job, I love the relationships you added ! 
How did you do those? Did you try using the `gen_relationships.py` tool on it? Happy to have feedback!

**Comment by adulau** [comment]
Thanks a lot for the update!

**Pull request merged**